modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -470,6 +471,47 @@ private void _createFDSFieldObjectDefinition(
 			ObjectRelationshipConstants.TYPE_ONE_TO_MANY, null);
 	}
 
+	private void _createFDSListSectionObjectDefinition(
+			ObjectDefinition fdsViewObjectDefinition, Locale locale,
+			long userId)
+		throws Exception {
+
+		ObjectDefinition fdsListSectionObjectDefinition =
+			_objectDefinitionLocalService.addSystemObjectDefinition(
+				"FDSListSection", userId, 0, "FDSListSection", "FDSListSection",
+				false, LocalizedMapUtil.getLocalizedMap("FDS List Section"),
+				true, "FDSListSection", null, null, null, null,
+				LocalizedMapUtil.getLocalizedMap("FDS List Sections"),
+				ObjectDefinitionConstants.SCOPE_COMPANY, null, 1,
+				WorkflowConstants.STATUS_DRAFT,
+				Arrays.asList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
+						_language.get(locale, "field-name"), "fieldName", true),
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
+						_language.get(locale, "name"), "name", true),
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, false, null,
+						_language.get(locale, "renderer-name"), "rendererName",
+						false)));
```

Comment:
The main idea for the new object entry `FDSListSection` is to represent building blocks of list visualization mode. In that way, it is equivalent to `FDSField`, the building block of table visualization mode. `FDSField` is really unfortunately named, it should be `FDSTableField` or `FDSTableColumn`. `FDSListSection` and `FDSField` are both entities that are 1:1 linked with `fieldName`, the main point of wiring to data definition (OpenAPI) and endpoints returned data. They are visual representation of schema property of a given `fieldName`. 

I added `rendererName` as optional attribute. We will not use it any time soon, but I am completely sure we will use it eventually. It serves to highlight why I chose this kind of data model.